finetuning.py

```diff
@@ -1,38 +1,49 @@
 import argparse
 
+import cv2
 from tensorflow.keras.optimizers import Adam
 
-from lr_face.data import EnfsiDataset
+from lr_face.data import LfwDevDataset
 from lr_face.losses import TripletLoss
 from lr_face.models import Architecture
 from lr_face.utils import fix_tensorflow_rtx
 from lr_face.versioning import Tag
 
 fix_tensorflow_rtx()
 
+AUGMENT_RESOLUTION = (50, 50)
```

Comment:
klopt het dat dit code block in deze PR zit? Lijkt me bij de andere horen? github issue?

Reply:
Nee is mijn fout, ik heropen dit PR zo even